src/S3IteratorLibsvm.cpp

```diff
@@ -80,355 +80,357 @@ std::shared_ptr<SparseDataset> S3IteratorLibsvm::getNext() {
   }
```

Comment:
Changes in this file are style-related, correct? Just to make sure.

Reply:
Yes. I do not recall editing this file's content.